src/app.rs

```diff
@@ -2,15 +2,15 @@
 
 use std::{error::Error as StdError, marker::PhantomData, path::Path, sync::Arc, time::Duration};
 
+use crate::shred::Resource;
```

Comment:
Why does shred in particular need to `crate::` prefix and not the other imports?

Reply:
Yeah that's kind of a funny little creation.  It's a result of the fact that our shred is pulled in here: https://github.com/amethyst/amethyst/blob/72021b6c0b39a0dee13755c1547a93845a398f2c/src/lib.rs#L92

Reply:
Ah I see.